babel.config.js

```diff
@@ -36,7 +36,7 @@ const config = {
             ]
         },
         test: {
-            plugins,
+            plugins: [...plugins, ['babel-plugin-dynamic-import-node']],
```

Comment:
This plugin is [required](https://jestjs.io/docs/en/webpack#using-with-webpack-2) for Jest to parse dynamic imports as requires.